watertap/costing/watertap_costing_package.py

```diff
@@ -197,6 +203,32 @@ def build_global_params(self):
             units=pyo.units.dimensionless,
         )
 
+        self.electrodialysis_total_membrane_cost = pyo.Var(
+            initialize=86,
+            doc="Electrodialysis membrane cost per cell pair is sum of costs for CEM and AEM",
+            units=self.base_currency / (pyo.units.meter**2),
+        )

```

Comment:
We discussed this.  But with a second thought I recommend more separating cem and aem.  The ion exchange membranes can be the same membrane with other units such as water electrolyzer. I am ok with keeping it as is.

Reply:
With regards to the 'specific energy calculation', I thought we had discussed this during our last chem team meeting and I thought that this was something that was automated by the costing package (i.e., the package can automatically get an energy cost per water produced). Maybe I am wrong on that. 

We can focus on getting your 1D PR merged first, then this can come after and I can make the necessary updates.